app/views/posts/index.html.erb

```diff
@@ -0,0 +1,37 @@
+
+<div class="user-post">
+
+    <div class="user-card">
+          <div class="user-image">
+              <img src="<%= @user.photo_link %>" alt="<%= @user.name %> photo" class="img-fluid rounded-circle">
+          </div>
+        <div class="user-info">
+          <h3><%= @user.name %></h3>
+          <p class="comment">Number of posts: <%= @user.posts.count %></p>
+       </div>
+    </div>
+      <% @posts.each_with_index do |post, index| %>
+      <div class="allposts-user">
+      
+       <h4><%= link_to "Post #{index + 1}", user_post_path(@user, post) %> </h4>
+      <p><%= post.text %></p>
+      <div class="likes">
+         <p class="comment">comments: <%= post.comments.count %></p>
+        <p class="likes"> likes: <%= post.likes.count %></p>
+      </div>
+    </div>  
+      <% end %>
+  <div class="all-comments">
+  <% @posts.each do |post| %>
+    <ul>
+      <% post.comments.each do |comment| %>
```

Comment:
- [ ] **[OPTIONAL]** The comments on the post index page should be the `recent_comments` in the Post model